DQMOffline/LumiZCounting/data/HLTFile_phys14

```diff
@@ -0,0 +1,98 @@
+// SingleMu
```

Comment:
Why not put these files into python configuration file?
I think it would cleaner that way (and less ways in which things can break: no file discovery, etc.)